rx-java-codegen/src/test/java/io/vertx/codegen/testmodel/EnhancedMethodsTCKImpl.java

```diff
@@ -0,0 +1,119 @@
+package io.vertx.codegen.testmodel;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class EnhancedMethodsTCKImpl implements EnhancedMethodsTCK {
+
+  @Override
+  public void methodWithOtherParam(Socket socket) {
+    System.out.println(socket.toString());
```

Comment:
these method should do something else than doing a println, i.e assertNotNull(socket),etc...

Reply:
Done